sandbox/registration-processor-dmz.properties

```diff
@@ -1,5 +1,8 @@
 ####################################################################################################
 #---------------------------------------- COMMON PROPERTIES ----------------------------------------
+
+registration.processor.main-process=NEW,UPDATE,LOST,RES_UPDATE,ACTIVATE,DEACTIVATE
+registration.processor.sub-process=BIOMETRIC_CORRECTION
```

Comment:
Do we need this property, can we remove this and simply assume the process not in main processses are additional info processes ?

Reply:
This Needs discussion with you and Monobikash,if the assumption is correct that all other processes are subprocess. because if we get any invalid process our system will start processing it as subprocess since we are not doing any vaidation anywhere

Reply:
Since this requires changes in many stages, it is planned to be done later, so revolved for now